t/t8015-blame-rename-detection.sh

```diff
@@ -0,0 +1,82 @@
+#!/bin/sh
+
+test_description='git blame rename detection control'
+
+. ./test-lib.sh
+
+test_expect_success 'setup test file rename with content changes' '
+	git init &&
+	echo abc >1.txt &&
+	echo def >>1.txt &&
+	echo ghi >>1.txt &&
+	git add . &&
+	git commit -m "Initial commit" &&
+
+	git mv 1.txt 2.txt &&
+	echo abc >2.txt &&
+	echo 123 >>2.txt &&
+	echo ghi >>2.txt &&
+	git add . &&
+	git commit -m "Rename+edit together"
+'
+
+# This test confirms that by default, git blame follows partial-file renames
+test_expect_success 'git blame follows inexact renames by default' '
+	FIXED_1=$(git rev-parse --short HEAD^) &&
+	FIXED_2=$(git rev-parse --short HEAD) &&
+
+	git blame 2.txt >output &&
+	grep "$FIXED_1" output | grep -q abc &&
+	grep "$FIXED_2" output | grep -q 123 &&
+	grep "$FIXED_1" output | grep -q ghi
+'
```

Comment:
This is too focused, and unnecessary. We really only need to verify that `1.txt` is mentioned only when the rename-detection score is high enough.